Add contribution philosophy and acknowledgments section

Asmita-das2502 opened the pull request.

Asmita-das2502 committed:
Add contribution philosophy and acknowledgments section

Roshanjossey commented:
I see that you've made changes to the wrong file.

Could you please add your name to https://github.com/firstcontributions/first-contributions/blob/master/Contributors.md ?

Asmita-das2502 commented:
Thank you for the feedback! I've updated the PR and added my name to the
Contributors.md file as requested. And   I sincerely apologize for the
oversight in making changes to the wrong file earlier. It was an honest
mistake, and I appreciate your patience in guiding me toward the correct
process.

On Sat, Jan 11, 2025 at 1:43 AM Roshan Jossy ***@***.***>
wrote:

> I see that you've made changes to the wrong file.
>
> Could you please add your name to
> https://github.com/firstcontributions/first-contributions/blob/master/Contributors.md
> ?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/firstcontributions/first-contributions/pull/92949#issuecomment-2583957423>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BFHYQLPNSY4CXGSEADM6V532KASWXAVCNFSM6AAAAABU4GYEPCVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDKOBTHE2TONBSGM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

Roshanjossey commented:
No worries. You don't have to apologize.

I've seen tutorial along the same lines as us suggesting to add something in Readme. I believe that's bad for maintainers of other projects who don't want such changes ( Most open source projects are like that). Hence I'm discouraging such changes.

I'm happy that you're learning from this. I wish you a pleasant journey in open source.|

Cheers

Asmita-das2502 commented:
Thanks a lot for understanding.

On Sat, Jan 11, 2025, 19:46 Roshan Jossy ***@***.***> wrote:

> No worries. You don't have to apologize.
>
> I've seen tutorial along the same lines as us suggesting to add something
> in Readme. I believe that's bad for maintainers of other projects who don't
> want such changes ( Most open source projects are like that). Hence I'm
> discouraging such changes.
>
> I'm happy that you're learning from this. I wish you a pleasant journey in
> open source.|
>
> Cheers
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/firstcontributions/first-contributions/pull/92949#issuecomment-2585283283>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BFHYQLNI6PKFWXVHSO6C6T32KERU5AVCNFSM6AAAAABU4GYEPCVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDKOBVGI4DGMRYGM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
